src/contexts/cart-context.js

```diff
@@ -1,11 +1,25 @@
-import React, { useContext, createContext, useState, useReducer, useEffect } from 'react'
-import { cartReducer } from '../reducer/cartReducer';
-import { getRequestDataFromServer } from '../utilities/helpers/api-calls-helper'
-
+import React, {
+  useContext,
+  createContext,
+  useState,
+  useReducer,
+  useEffect
+} from 'react'
+import {
+  cartReducer
+} from '../reducer/cartReducer';
+import {
+  getRequestDataFromServer
+} from '../utilities/helpers/api-calls-helper'
```

Comment:
you can create an index.js file where you export all files. by this, you only have to write  `from "reducer"` .